pkg/openfeature/client_test.go

```diff
@@ -660,33 +753,4 @@ func TestErrorCodeFromProviderReturnedInEvaluationDetails(t *testing.T) {
 			generalErrorCode, evalDetails.ErrorCode,
 		)
 	}
-}
-
-func TestValueUsedFromProvider(t *testing.T) {
-	defer t.Cleanup(initSingleton)
-	ctrl := gomock.NewController(t)
-
-	mockProvider := NewMockFeatureProvider(ctrl)
-	mockProvider.EXPECT().Metadata().AnyTimes()
-	mockProvider.EXPECT().Hooks().AnyTimes()
-	SetProvider(mockProvider)
-	mockProvider.EXPECT().BooleanEvaluation(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).
-		Return(BoolResolutionDetail{
-			Value: true,
-			ProviderResolutionDetail: ProviderResolutionDetail{
-				Reason: TargetingMatchReason,
-			},
-		})
-
-	client := NewClient("test")
-	evalDetails, err := client.evaluate(
-		context.Background(), "foo", Boolean, false, EvaluationContext{}, EvaluationOptions{},
-	)
-	if err != nil {
-		t.Errorf("Unexpected Exception occurred: %s", err.Error())
-	}
-
-	if evalDetails.Value != true {
-		t.Error("Expected true value, but got default false.")
-	}
-}
```

Comment:
@rgrassian-split  there was nothing at all wrong with your test, but it's redundant to the missing 1.4.2 ones, so I've removed it in favor of those (they are pretty much identical, just cover all the resolver methods)